Laiban/Modules/Recreation/Sources/Admin/AdminRecreationAddInventoryItemView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by jonatan lidholm jansson on 2022-10-19.
+//
+
+import SwiftUI
+
+struct AdminRecreationAddInventoryItemView: View {
+    var service:RecreationService
+    var inventoryType:InventoryType
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @State var newInventoryItem: Recreation.Inventory.Item = .init()
+    @State var emoji:String = String()
+    var body: some View {
+        Form {
+            Section{
+                TextField("Inventarie", text:$newInventoryItem.name)
```

Comment:
We may consider using something other than 'Inventarie' as name for this. Not sure what we should use, maybe 'föremål' eller  'objekt' (we're using 'föremål' in AdminRecreationActivityListView)?